src/main/java/com/github/cameltooling/lsp/internal/instancemodel/OptionParamURIInstance.java

```diff
@@ -90,6 +93,12 @@ public String getDescription(ComponentModel componentModel, KameletsCatalogManag
 		EndpointOptionModel model = componentModel.getEndpointOption(keyName);
 		if(model != null) {
 			return model.getDescription();
+		} else if (componentModel.getScheme().equalsIgnoreCase("kamelet")) {
```

Comment:
The constant ComponentNameConstants.COMPONENT_NAME_KAMELET can be used https://github.com/camel-tooling/camel-language-server/blob/9445a7198d6d64abb3197d3580785e6d7722219e/src/main/java/com/github/cameltooling/lsp/internal/instancemodel/ComponentNameConstants.java#L22